app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
Comparisons to the singleton objects, like True, False, and None, should be done with identity, not equality. Use “is” or “is not”.
It will be correct like this:  # if "vaccine" not in vstr.keys() or vstr["vaccine"] is None: